content/ember-4-5-released.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Ember 4.5 Released
+authors:
+  - jen-weber
+date: 2022-06-20T00:00:00.000Z
+tags:
+  - releases
+  - '2022'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.5 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 4.5 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 4.5
+
+Ember.js 4.5 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 4.5 introduced 0 bug fixes.
+
+#### Features
+
+Ember.js 4.5 introduced 2 new features.
+
+1. Plain function as helpers
+2. `renderSettled` is now public API
+
+##### 1. Plain functions as helpers
+
+You can now use plain functions as helpers in your component templates. This helps make the relationship between Ember component templates and their JavaScript class more intuitive.
+The full documentation can be found in [TODO link to guides]()
```

Comment:
Maybe let's remove this sentence from here, and then add some text below after the example?

```suggestion
```